wiki/osu!_Program_Files/User_configuration_file/fr.md

```diff
@@ -0,0 +1,230 @@
+
+***[Click to return to Main page](Osu!_Program_Files "wikilink")***
+
+<img src="CF_DEAD.jpg" title="fig:Combo Fire." alt="Combo Fire." width="200" /> Ceci est une liste d'option située dans le fichier osu!.<nomdevotrepc>.cfg. Beaucoup de ces options son disponibles en jeu et il est recommandé d'y modifier ces options par son biais. N'éditez ces valeurs manuellement qu'en cas de vrai besoin.
```

Comment:
you also need to escape the less than and greater than signs
`osu!.\<nomdevotrepc\>.cfg` otherwise GitHub (and live osu!wiki?) will think it is an actual HTML tag and will not display it